src/fsharp/vs/Symbols.fs

```diff
@@ -1397,7 +1397,29 @@ and FSharpMemberOrFunctionOrValue(cenv, d:FSharpMemberOrValData, item) =
 
         | V v -> 
         match v.ValReprInfo with 
-        | None -> failwith "not a module let binding or member"
+        | None ->
+            let _, tau = v.TypeScheme
+            if isFunTy cenv.g tau then
+                let typeArguments, _typ = stripFunTy cenv.g tau
+                [ for typ in typeArguments do
+                    let allArguments =
+                        if isTupleTy cenv.g typ
+                        then tryDestTupleTy cenv.g typ
+                        else [typ]
+                    yield
+                      [for typ in allArguments do
+                          match typ with
+                          | TType_var tp ->
```

Comment:
The match also serves to get the `typar` `tp` which is the second param of `FSharpParameter`.  The attributes can be removed as the spec doesn't support them does it?  I wasn't positive so...